rtfd/rtfd.py

```diff
@@ -44,10 +44,20 @@ def title_scraper(query):
     source_code = requests.get(url).text
     soup = BeautifulSoup(source_code, 'html.parser')
     for p in soup.find_all('li', {'class': 'module-item'}):
-        title_p, desc_p, *_ =  p.find_all('p')
+        ps = p.find_all('p')
+
+        try:
+            title_p, desc_p, *_ = ps
+        except ValueError:
+            # some results have no description
+            title_p = ps[0]
+            desc = ''
```

Comment:
Using "Description not available." would be better rather than just a blank space.